src/encoded/schemas/page.json

```diff
@@ -86,6 +88,12 @@
                     "type": "boolean",
                     "default": false
                 }
+            },
```

Comment:
Edits in this page.json file are unrelated to task/branch. Adds default values for "table-of-contents" property (enabled : false) for data-consistency/fullness.